misc/scripts/build.sh

```diff
@@ -140,6 +139,9 @@ else
     -e "ESX=$2" \
     -e "VM1=$3" \
     -e "VM2=$4" \
+    -e "DS1=$DS1" \
+    -e "VM1NAME=$VM1NAME" \
```

Comment:
Does this issue happen on 6.0U2 and 6.5? Can you please file an issue for this? We need to make sure the test utils work on all supported platforms. @shuklanirdesh82 

Our goal is to keep the number of shared environment variables to the minimum (and use common utils to get whatever we need), because these variables must be defined when we run e2e tests locally. If there are too many variables to be defined, it will be too cumbersome to run e2e tests locally.